packages/cosmic-swingset/src/chain-main.js

```diff
@@ -90,6 +90,19 @@ const makeBootMsg = initAction => {
   };
 };
 
+/**
+ * Extract local Swingset-specific configuration which is
+ * not part of the consensus.
+ *
+ * @param {any} initAction
+ */
+const makeSwingsetConfig = initAction => {
+  const { maxVatsOnline = 50 } = initAction.resolvedConfig || {};
```

Comment:
Do we want this default specified in two places? I'm wondering if we can just let it be undefined here.

Reply:
should we also remove one from [vat-warehouse.js] (https://github.com/Agoric/agoric-sdk/blob/31eda7309df378d0612455ddb723711ec6e710eb/packages/SwingSet/src/kernel/vat-warehouse.js#L246) ?

Reply:
the default in `vat-warehouse` should likely stay, but maybe we should cross reference them to keep them in sync. That said once a default is written into a `toml` file, changing the default anywhere (cosmos or here) will not change the value until validators would regenerate their config (not a normal process)